blocks/api/post.pegjs

```
@@ -1,15 +1,11 @@
 {
 
-function untransformValue( value ) {
-	return 'string' === typeof value
-		? value.replace( /\\-/g, '-' )
-		: value;
-}
-
-function keyValue( key, value ) {
-  const o = {};
-  o[ key ] = untransformValue( value );
-  return o;
+function maybeJSON( s ) {
```

Can we report an error somehow?